.build/Build.ps1

```diff
@@ -139,7 +139,14 @@ if ($nonUnique.Count -gt 0) {
 
 $scriptVersions = @()
 
-$disclaimer = [IO.File]::ReadAllLines("$PSScriptRoot\..\LICENSE")
+if ($IsMacOS -or
+    $IsLinux) {
+    Write-Host "We're running the build process on MacOS / Linux"
+    $disclaimer = [IO.File]::ReadAllLines("$((Get-Location).Path)/../LICENSE")
+} else {
+    Write-Host "We're running the build process on Windows"
+    $disclaimer = [IO.File]::ReadAllLines("$PSScriptRoot\..\LICENSE")
+}
```

Comment:
```suggestion
$disclaimer = [IO.File]::ReadAllLines([IO.Path]::Combine($PSScriptRoot, "..", "LICENSE"))
```

Suggest replacing all of this with an environment-agnostic path combine. If we really want to call out what OS we're running on, I would like to separately call out all three, not combine MacOS and Linux together, since they are not the same.